pkg/machinepool/machinepool_test.go

```diff
@@ -385,8 +385,8 @@ var _ = Describe("Machinepool and nodepool", func() {
 			args := ListMachinePoolArgs{ShowAll: true}
 			out := getMachinePoolsString(r, cluster.MachinePools().Slice(), args)
 
-			expectedOutput := "ID\tAUTOSCALING\tREPLICAS\tINSTANCE TYPE\tLABELS\tTAINTS\tAVAILABILITY ZONES\tSUBNETS\tSPOT INSTANCES\tDISK SIZE\tSG IDS\tAZ TYPE\tWIN-LI ENABLED\tDEDICATED HOST\n" +
-				"mp-1\tNo\t3\tm5.large\t\t\t\t\tNo\tdefault\t\tN/A\tNo\tNo\n"
```

Comment:
I'm a little confused, the test name is `Test printMachinePools with showAll flag` - shouldn't it show all 3 columns instead of hiding them?

Reply:
Also we should update the test name no matter what to have `--all` instead of `showAll` if you don't mind 🙂